scripts/pull-kinto-admin.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+VERSION=$(cat kinto/plugins/admin/VERSION)
+TAG="v${VERSION}"
+
+# download and unzip release
+curl -OL "https://github.com/Kinto/kinto-admin/releases/download/${TAG}/kinto-admin-release.tar"
+rm -r ./kinto/plugins/admin/build || echo "admin/build folder doesn't exist yet"
```

Comment:
```suggestion
rm -rf ./kinto/plugins/admin/build
```

Or is this too dangerous?

Reply:
If this script was only used for docker I'd be fine with the `-f` but the echo makes me feel better when running it through `make` on my machine. I don't think it's really dangerous though.